spec/requests/admin/feedback_spec.rb

```diff
@@ -39,7 +39,7 @@
 
       it "does not show navigation links" do
         subject
-        expect(parsed_response_body.css(".pagination-container nav")).to be_empty
+        expect(parsed_response_body.css(".govuk-pagination")).to be_empty
```

Comment:
While you're here, how do you feel about updating this to use Capybara matchers?

```ruby
expect(page).not_to have_css(".govuk-pagination")
```